schemas/com.apple.swiftui/immersive_space/jsonschema/1-0-0

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "com.apple.swiftui",
+    "name": "immersive_space",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Schema for an immersive space entity, representing the VisionOS immersive space that the event occurs in.",
+  "type": "object",
+  "properties": {
+    "view_id": {
+      "type": ["string", "null"],
+      "format": "uuid",
+      "description": "UUID for the view of the immersive space."
+    },
+    "id": {
+      "type": "string",
```

Comment:
One more thing before merging this into the release branch – could you add a max length here? We usually go with something like 255, 1024, 4096 – I think 255 would be good.